### PR TITLE
Feat/skeleton

### PR DESCRIPTION
**Pull request by Jesusborjas006**
Change Summary: This branch organizes the projects file structure by adding all components to a single folder. Each folder contains it's own tsx and css file. Skeleton for application is created with HTML, CSS and made use of Material UI to add a button to the form card. This is just the start of the skeleton and there is yet more elements to be added.

### CODE REVIEW

**Review by kirch1 — COMMENTED** [review]
Nice wireframe starting point. Think we are ready to start adding state and api calls.

File: src/Components/Results/Results.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  return (
+    <div className="results-container">
+      <div className="list-content">
+        <ul className="list">
+          <li>Privacy</li>
+          <li>Security</li>
+          <li>Intellectual property</li>
+          <li>Liability and indemnification</li>
+          <li>Termination and cancellation</li>
+          <li>Payment and fees</li>
```
Comment (chrissycooper) [inline src/Components/Results/Results.tsx:13]: Thank you for putting these in!!

Comment (chrissycooper) [reply src/Components/Results/Results.tsx:13]: googling 'indemnification' rn! I wonder if we want to capitalize each word (besides `and`), so e.g. `Intellectual Property`

File: src/index.css [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
+  margin: 0;
+  padding: 0;
 }
 
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
+    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
```
Comment (chrissycooper) [inline src/index.css:10]: nice attention to detail!! Appreciate you

**Review by chrissycooper — APPROVED** [review]
Looks great, thank you!

**Pull request merged**